packages/devtools/lib/devtools.dart

```diff
@@ -5,4 +5,4 @@
 /// The DevTools application version.
 // Note: when updating this, please update the corresponding version in the
 // pubspec.
-const String version = '0.0.13-dev.1';
+const String version = '0.0.14-dev.1';
```

Comment:
did you mean for this to be a dev release or a regular release?

Reply:
I think a regular release I'll remove the -dev.1

Reply:
It could be `0.0.13` (since `0.0.13-xxx` is a pre-release version of 0.0.13). It's not really important here, but if we release as 0.0.14 and then make another dev version, it should be 0.0.15-dev (since 0.0.14-dev is less than 0.0.14).

(If it wasn't clear, the difference with pre-release versions is that `pub global activate devtools` won't pick pre-release versions unless the version is explicitly provided.. If we want users to get the update, it needs to be a regular release)

Reply:
I'll make it 0.0.13 thanks Danny.